graph/community/louvain_undirected_test.go

```diff
@@ -153,13 +153,16 @@ var communityUndirectedQTests = []communityUndirectedQTest{
 				},
 			},
 			{
-				q: 0.39907955292570674,
+				q: 0.3496877054569362,
 				communities: [][]graph.Node{
 					{simple.Node(0), simple.Node(1), simple.Node(2), simple.Node(3), simple.Node(7), simple.Node(11), simple.Node(12), simple.Node(13), simple.Node(17), simple.Node(19), simple.Node(21)},
 					{simple.Node(4), simple.Node(10)},
 					{simple.Node(5), simple.Node(6), simple.Node(16)},
-					{simple.Node(8), simple.Node(9), simple.Node(14), simple.Node(15), simple.Node(18), simple.Node(20), simple.Node(22), simple.Node(26), simple.Node(29), simple.Node(30), simple.Node(32), simple.Node(33)},
-					{simple.Node(23), simple.Node(24), simple.Node(25), simple.Node(27), simple.Node(28), simple.Node(31)},
+					{simple.Node(8), simple.Node(9), simple.Node(14), simple.Node(15), simple.Node(18), simple.Node(20), simple.Node(22), simple.Node(30), simple.Node(32), simple.Node(33)},
+					{simple.Node(23), simple.Node(25)},
```

Comment:
It's not a rewrite, I needed to change intermediate community reduction results to get the test to pass. This is OK because the algorithm is randomised and so doesn't guarantee the same results and more significantly the intermediate result shown here are not necessarily meaningful as a community modularisation.

Reply:
Great. Just wanted to check.